README.md

```diff
@@ -1,18 +1,32 @@
-# cli
+# Welcome to urfave/cli
 
 [![Run Tests](https://github.com/urfave/cli/actions/workflows/cli.yml/badge.svg)](https://github.com/urfave/cli/actions/workflows/cli.yml)
 [![Go Reference](https://pkg.go.dev/badge/github.com/urfave/cli/v3.svg)](https://pkg.go.dev/github.com/urfave/cli/v3)
 [![Go Report Card](https://goreportcard.com/badge/github.com/urfave/cli/v3)](https://goreportcard.com/report/github.com/urfave/cli/v3)
 [![codecov](https://codecov.io/gh/urfave/cli/branch/main/graph/badge.svg?token=t9YGWLh05g)](https://codecov.io/gh/urfave/cli)
 
-cli is a simple, fast, and fun package for building command line apps in Go. The
-goal is to enable developers to write fast and distributable command line
-applications in an expressive way.
+urfave/cli is a **declarative**, simple, fast, and fun package for building command line tools in Go featuring:
+
+- nestable commands with alias and prefix match support
+- flexible and permissive help system
```

Comment:
Permissive help? What does it permit?

Reply:
I was thinking of how the default behavior allows `help` before and after positional commands, and has a default `h` alias, and also the `--help` `-h` flags, although these behave in a more standard way. I picked "permissive" because I wanted a single word that describes how the help system is trying to be compatible with various usage styles.

Reply:
The ordering is true for the flag form, but the behavior is a little different for the command form which treats adjacent command name and `help` as equivalent no matter the order, meaning these are equivalent:

```
prog cmd help
```

```
prog help cmd
```